CONTRIBUTING.md

```diff
@@ -10,6 +10,17 @@ New **NPM** releases are not tracked. We simply use the NPM version bundled in t
 
 **Yarn** is updated to the latest version only when there is a new Node.js SemVer PATCH release (unless Yarn has received a security update), and it's updated only in the branch with the new release, preferably in the same PR. The `update.sh` script does this automatically when invoked with a specific branch, e.g. `./update.sh 6.10`.
 
+### Submitting a PR for a version update
+
+If you'd like to help us by submitting a PR for a version update, please do the following:
+
+1. [Fork this project.](https://help.github.com/en/github/getting-started-with-github/fork-a-repo)
+2. [Clone the forked repository.](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository)
+3. Create a branch for the update PR. For example, `git checkout master; git checkout -b version-update`.
+4. Run `./update.sh`. You can see additional options by using accessing the built-in help documentation with `./update.sh -h`. This script will automatically update the appropriate files with the latest versions and checksums.
+5. Commit the modified files to the `version-update` branch and push the branch to your fork.
+6. [Create a PR to merge the branch from your fork into this project's master branch.](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork).
```

Comment:
I suggest to use `1. ` only, so we don't need to maintain the numbers.

Reply:
Ya know, I prefer that myself, but my markdown plugin "helped me" by changing them all to sequential numbers. I'm happy to update it with `1.`.

Reply:
Done, resolving. Thanks!